packages/clerk-js/src/core/resources/SessionTask.ts

```diff
@@ -0,0 +1,61 @@
+import type {
+  ClerkOptions,
+  EnvironmentResource,
+  SessionTaskJSON,
+  SessionTaskJSONSnapshot,
+  SessionTaskKey,
+  SessionTaskResource,
+} from '@clerk/types';
+
+import { buildURL, inBrowser } from '../../utils';
+
+export const SESSION_TASK_PATHS = ['add-organization'] as const;
+type SessionTaskPath = (typeof SESSION_TASK_PATHS)[number];
+
+export const SESSION_TASK_PATH_BY_KEY: Record<SessionTaskKey, SessionTaskPath> = {
+  org: 'add-organization',
```

Comment:
Idea: comment here explaining that when we add a new task type, we also need to add a corresponding resolution url, enforced by the compiler